src/Source/FileSource.js

```diff
@@ -145,10 +146,32 @@ class FileSource extends Source {
         return this.url;
     }
 
-    onParsedFile(parsedFile) {
```

Comment:
Can a retrocompatibility be added for a few version ?

Reply:
retrocompatibility added